sktime/series_as_features/base/estimators/tests/test_feature_importances_.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pytest
+
+from sklearn.base import clone
+from sklearn.pipeline import Pipeline
+from sklearn.pipeline import FeatureUnion
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.preprocessing import FunctionTransformer
+from sktime.datasets import load_gunpoint
+from sktime.utils.time_series import time_series_slope
+from sktime.transformers.series_as_features.segment import IntervalSegmenter
+from sktime.transformers.series_as_features.compose import RowTransformer
+from sktime.transformers.series_as_features.summarize._extract import \
+    RandomIntervalFeatureExtractor
+from sktime.classification.compose._ensemble import TimeSeriesForestClassifier
+
+X_train, y_train = load_gunpoint(split="TRAIN", return_X_y=True)
```

Comment:
```suggestion
X_train, y_train = make_classification_problem()
```
You can import it from `sktime.utils.testing`, that may speed up the tests a bit